api/rest/v1/bulk_info_variable.md

```diff
@@ -0,0 +1,484 @@
+---
+layout: default
+title: Info on a variable
+nav_exclude: true
+parent: v1 REST
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/info/variable
+---
+
+
+
+## /v1/bulk/info/variable
+
+Get basic information about multiple [variables](/api/rest/v1/getting_started#variable).
+
+This API returns basic information on multiple variables, given each of their [DCIDs](/api/rest/v1/getting_started#dcid). The information is provided per variable, and includes the number of places with data on each variable, the minimum and maximum values observed, and the name and DCID of the top 3 entities with highest observed values for each variable.
```

Comment:
You can try some markdown lint tool to break out the long lines

Reply:
Done!